features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -260,6 +260,7 @@ CREATE TABLE IDN_AUTH_SESSION_STORE (
             OPERATION VARCHAR(10) NOT NULL,
             SESSION_OBJECT BLOB,
             TIME_CREATED BIGINT NOT NULL,
+            EXPIRY_TIME BIGINT NOT NULL,
```

Comment:
Can we add this column to the end of the table? This might introduce other problem if there are select * queries within the code. On the other hand, it's difficult to migrate with db2 and some other databases as they don't support AFTER syntax with ALTER.